### PR TITLE
[bitnami/maradb-galera] Added extraVolumes and extraVolumeMounts

### PR DESCRIPTION
**Pull request by luizgn**
**Description of the change**

Allow mount of extra volumes.

**Benefits**

Using extra volumes, it is possible to enable data-in-transit encryption and data-at-rest encryption using file plugin from secrets.

**Possible drawbacks**

There is no drawbacks.

**Applicable issues**

There is no issues identified.

**Additional information**

Tested with Helm v3.0.2 and Kubernetes v1.15.6.

**Checklist** 
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/mariadb-galera/README.md [hunk 1]
```diff
@@ -130,6 +130,8 @@ The following table lists the configurable parameters of the MariaDB Galera char
 | `persistence.size`                   | Persistent Volume Size                                                                                                                                      | `8Gi`                                                             |
 | `extraInitContainers`                | Additional init containers (this value is evaluated as a template)                                                                                          | `[]`                                                              |
 | `extraContainers`                    | Additional containers (this value is evaluated as a template)                                                                                               | `[]`                                                              |
+| `extraVolumes`                       | Extra volumes                                                                                                                                               |                                                                   |
```
Comment (sameersbn) [inline bitnami/mariadb-galera/README.md]: Please add `nil` to the default column.

File: bitnami/mariadb-galera/README.md [hunk 2]
```diff
@@ -130,6 +130,8 @@ The following table lists the configurable parameters of the MariaDB Galera char
...
+| `extraVolumeMounts`                  | Mount extra volume(s)                                                                                                                                       |                                                                   |
```
Comment (sameersbn) [inline bitnami/mariadb-galera/README.md]: Please add `nil` to the default column.

File: bitnami/mariadb-galera/README.md (hunk 1 above)
Comment (luizgn) [reply bitnami/mariadb-galera/README.md]: Changed.

File: bitnami/mariadb-galera/README.md (hunk 2 above)
Comment (luizgn) [reply bitnami/mariadb-galera/README.md]: Changed.

**Pull request merged**